plugins/convert_README_to_header.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
+
+# A simple script to build a C++ header file containing the plugin description
+# from the plugin's README.md
+# The output file then contains one variable 'plugin_description'.
+
+if [ $# != 2 ]; then
+  echo "Usage: convert_README_to_header.sh <path> <dest file>";
+  echo "<path>: path to plugin dir, e.g. plugins/artnet";
+  echo "<dest file>: e.g. include/ola/plugins/artnet_description.h";
+  exit 1;
+fi
+
+path="$1";
+outfile="$2";
+
+if [ ! -d $path ]; then
+  echo "directory '$path' does not exist";
+  exit 1;
+fi
+
+if [ ! -e "$path/README.md" ]; then
+  echo "README.md file in '$path' does not exist";
+  exit 1;
+fi
+
+plugin=`basename "$path"`;
+outfilename=`basename $outfile`;
+
+# See http://stackoverflow.com/a/16576291
+# On Mac OS's sed, \n is not recognized as a newline character, but
+# \[actual newline] works
+desc=`sed -e ':a' -e 'N' -e '$!ba' -e 's/\n/\\\\n"\\
+"/g' "$path/README.md"`;
+
+identifier="PLUGINS_${plugin}_PLUGIN_DESCRIPTION_H_";
```

Comment:
Actually, because I don't know if I should stick to the line length limit in shell scripts, too. So I just bypassed the issue ;)